next.data.mjs

```diff
@@ -1,21 +1,20 @@
 import * as preBuild from './scripts/next-data/generatePreBuildFiles.mjs';
 
-import getNodeVersionData from './scripts/next-data/getNodeVersionData.mjs';
 import getBlogData from './scripts/next-data/getBlogData.mjs';
```

Comment:
Can I ask a little favour here?

Can you make an `index.mjs` on `scripts/next-data`, importing all the default exports from these next-data scripts... And then we can simply:

```tsx
import nextData from './scripts/next-data`;

const cachedBlogData = nextData.getBlogData();

nextData.generateNodeReleasesData();
...
```

Reply:
Sure, I think it's neat.